src/draw.c

```diff
@@ -412,12 +412,12 @@ static int layout_get_height(struct colored_layout *cl)
  */
 static int frame_internal_radius (int r, int w, int h)
 {
-        if (r == 0 || w == 0 || h == 0)
-                return 0;
-
         // Integer precision scaler, using 1/4 of int size
         const int s = 2 << (8 * sizeof(int) / 4);
 
+        if (r == 0 || r == w || s == 0 || h == - (w - r) * 2)
```

Comment:
```suggestion
        if (r == 0 || h + (w - r) * 2 == 0)
```

Some points:
* `s == 0` is indeed overkill, it's a constant after all.
* What's the point of `r == w`? The final check already checks for a division by 0 in that equation.

And a small readability nitpick, compare everything with 0 to be clear what it's actually checking.

Reply:
You are absolutely right!  
I first thought of `r == w` then I completed with invalid values for h without remembering to get rid of the first test.

Reply:
@MaximeWack ping? No pressure if you're busy, but I'm waiting for you to update the PR in order to merge this.

Reply:
Oops, sorry, forgot about this ! It's updated now.

Thank you !